style/message-proposal-1.css

```diff
@@ -16,6 +16,50 @@ body {
   line-height: 1.2em;
 }
 
+nav#gutter-top {
+  position:              fixed;
+  top:                   0;
+  left:                  0;
+  right:                 0;
+  height:                53px;
+  background-color:      #2975ac;
+  border:                none;
+  font-size:             larger;
+  color:                 #e0e0e0;
+}
+
+nav#gutter-top > .container-fluid {
+  padding-left:          0;
+}
+
+.navbar-default .navbar-nav > li > a {
+  color:                 #e0e0e0;
+}
+.navbar-default .navbar-nav > li > a:focus,
+.navbar-default .navbar-nav > li > a:hover,
+.navbar-default .navbar-nav > li > a:active {
+  color:                 #ffffff;
+}
+
+.navbar-default .navbar-nav li:first-child a {
+  padding:               0;
+}
+
+div#main {
+  margin-top:            73px;
+}
+
+div#overlay {
+  position:              absolute;
+  top:                   0;
+  right:                 0;
+  min-width:             5em;
+  padding:               0.1em 0.5em;
+  border-color:          #eee;
+  border-style:          solid;
+  border-width:          0 0 2px 2px;
+}
+
```

Comment:
I think a consistent codebase is key. So in my opinion it would be better if you’d stick to the already used formatting convention rather than using yours here. It’s not meant offensive in any way, it’s just confusing to me to have different formats in one stylesheet.